src/types/YText.js

```diff
@@ -737,6 +738,17 @@ export class YText extends AbstractType {
     return this._length
   }
 
+  /**
+   * Permit empty paragraph at the end of the content when applyDelta.
+   *
+   * @param {boolean} permit
+   *
+   * @public
+   */
+  set permitEmptyParagraph (permit) {
```

Comment:
This should be an option that you pass to `applyDelta`. I.e. `ytext.applyDelta(delta, { sanitize: true })`

Where applyDelta should be declared as follows:

```js
  /**
   * Apply a {@link Delta} on this shared YText type.
   *
   * @param {any} delta The changes to apply on this element.
   * @param {object}  [opts]
   * @param {boolean} [opts.sanitize] Sanitize input delta. Removes ending newlines if set to true.
   *
   * @public
   *
applyDelta (delta, { sanitize = true } = {}) {
 ..
}
```